resource/global/YoStarEN/resource/tasks.json

```diff
@@ -291,5 +291,906 @@
+		"Doc": "该任务的 ocrReplace 被所有涉及Rouge-like的English识别任务复用",
+        "ocrReplace": [
+            [
+                "Beast Taming",
+                "驯兽小屋"
+            ],
+            [
+                "Accident",
+                "意外"
+            ],
+            [
+                "A Date With Slugs",
+                "与虫为伴"
+            ],
+            [
+                "Gun Salute",
+                "礼炮小队"
+            ]
+        ],
+        "roi": [
+            250,
+            435,
+            800,
+            100
+        ]
+    },
+    "CharsNameOcrReplace": {
+        "algorithm": "OcrDetect",
+        "text": [],
+        "Doc": "该任务的 ocrReplace 被所有涉及干员名的English Name识别任务复用",
+        "ocrReplace": [
+            [
+                "SilverAsh",
+                "银灰"
+            ],
+            [
+                "Blaze",
+                "煌"
+            ],
+            [
+                "Surtr",
+                "史尔特尔"
+            ],
+            [
+                "Ch'en",
+                "陈"
+            ],
+            [
+                "Thorns",
+                "棘刺"
+            ],
+            [
+                "Specter",
+                "幽灵鲨"
+            ],
+            [
+                "Nearl the Radiant Knight",
+                "耀骑士临光"
+            ],
+            [
+                "Lappland",
+                "拉普兰德"
+            ],
+            [
+                "Indra",
+                "因陀罗"
+            ],
+            [
+                "Amiya",
+                "阿米娅"
+            ],
+            [
+                "Broca",
+                "布洛卡"
+            ],
+            [
+                "Savage",
+                "暴行"
+            ],
+            [
+                "Bibeak",
+                "柏喙"
+            ],
+            [
+                "Franka",
+                "芙兰卡"
+            ],
+            [
+                "Sideroca",
+                "铸铁"
+            ],
+            [
+                "Swire",
+                "诗怀雅"
+            ],
+            [
+                "Whislash",
+                "鞭刃"
+            ],
+            [
+                "Akafuyu",
+                "赤冬"
+            ],
+            [
+                "Flamebringer",
+                "炎客"
+            ],
+            [
+                "Melantha",
+                "玫兰莎"
+            ],
+            [
+                "Dobermann",
+                "杜宾"
+            ],
+            [
+                "Matoimaru",
+                "缠丸"
+            ],
+            [
+                "Frostleaf",
+                "霜叶"
+            ],
+            [
+                "Estelle",
+                "艾丝黛尔"
+            ],
+            [
+                "Hellagur",
+                "赫拉格"
+            ],
+            [
+                "Pallas",
+                "帕拉斯"
+            ],
+            [
+                "Mountain",
+                "山"
+            ],
+            [
+                "Skadi",
```

Comment:
Here is the regex replacement, `Skadi` will interfere with `Skadi the Corrupting Heart`, pls use `^Skadi$` instead of it.

Reply:
Same for `Nearl`, `Ch'en`, etc.